app.py

```diff
@@ -13,10 +13,25 @@
 # Create App and stacks
 app = core.App()
 
+artifact_bucket = app.node.try_get_context("ArtifactBucket")
+
 # Create the pipeline stack
-PipelineStack(app, "drift-pipeline")
+synth = core.DefaultStackSynthesizer()
+if artifact_bucket is not None:
+    synth = core.DefaultStackSynthesizer(
+        file_assets_bucket_name=artifact_bucket,
+        generate_bootstrap_version_rule=False,
+    )
+PipelineStack(app, "drift-pipeline", synthesizer=synth)
 
 # Create the SC stack
-ServiceCatalogStack(app, "drift-service-catalog")
+synth = core.DefaultStackSynthesizer()
```

Comment:
Do we need to declare this twice, or can we use the synth object?

Reply:
We need to declare it for every synth operation, we can't reuse it after "spending" it